### ISSUE TITLE
Instruction of TVM COV

### ISSUE THREAD

**Issue by shaoyuyoung** [ise-uiuc/nnsmith#140]
Hi, Jiawei @ganler . I'm sorry to bother you due to your busy schedule. I have some difficulty in collecting `branch coverage of TVM`. 

I followed your instructions for [Torch Cov](https://github.com/ise-uiuc/nnsmith/tree/main/experiments#readme) and it worked. 

 I found that [Tzer](https://github.com/ise-uiuc/tzer) provides an [instruction of TVM Memcov](https://github.com/ganler/memcov/tree/70e901637e718646c548fffae23a68152df7c6d1). I followed this, but unfortunately, it didn't work when I used the [experiments scripts](https://github.com/ise-uiuc/nnsmith/tree/main/experiments) NNSmith provides. I couldn't get the profraw files.

I think this is due to the fact that the [patch file](https://github.com/ise-uiuc/tzer/blob/main/tvm_cov_patch/memcov4tvm.patch) is not specifically designed for `branch cov` but for `edge cov`.  Do you still have a patch for `branch cov` for TVM? (or maybe some tutorials to help me make the patch)

Thx in advance :D

**Comment by shaoyuyoung** [comment]
I have solved this problem.

This is still misleading because of unclear instructions :(

When I have free time, I can contribute to adding the instruction of TVM like [Torch](https://github.com/ise-uiuc/nnsmith/tree/main/experiments)

- [ ] Add the instruction of TVM

**Comment by ganler** [comment]
Hello @shaoyuyoung 

Memcov is CFG coverage. In NNSmith and NeuRI I chose source code coverage.

To understand the differences between the two, you can check out my earlier blog: https://jw-liu.xyz/blog/2022/what-we-talk-when-we-talke-about-coverage/

TL;DR: For coverage evaluation, use source-code coverage cuz it's accurate; for coverage-guided fuzzing, use CFG coverage cuz it's fast.

**Comment by ganler** [comment]
> is not specifically designed for branch cov but for edge cov

FYI, edge coverage is actually more accurate than branch coverage. i.e., larger search space. :)

**Comment by shaoyuyoung** [comment]
thx for your answer :)
